src/components/autocomplete.js

```diff
@@ -82,17 +81,22 @@ const Autocomplete = () => {
       <InputGroup className="mb-3">
         <Input
           type="text"
-          placeholder={'Enter your breed'}
+          placeholder="Enter your breed"
           autoComplete="off"
-          onChange={(e) => changeHandler(e.target.value)}
+          onChange={(e) => setInput(e.target.value)}
           onKeyDown={keyDownHandler}
           value={input}
-          aria-label="Recipient's username"
+          aria-label="Search by Breed Name"
           aria-describedby="search"
         />
         <InputSearch id="search">
-          <Link to={`/breed/${input}`}>
-            <Search style={{ fill: colors.dark }} />
+          <Link to={input ? `/breed/${input}` : ''}>
+            <Search
+              style={{
+                fill: input ? colors.dark : colors.darkGrey,
+                cursor: input ? 'pointer' : 'not-allowed'
+              }}
```

Comment:
The style object is new on each rerender. Generally, we don't use inline styles.